.eslintrc.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 12,
   },
-  ignorePatterns: ['/*.js'],
+  ignorePatterns: ['!.*', '!.circleci/test-batch.js'],
```

Comment:
Is this right?  I'm reading it as "not ignoring all files in this folder and not ignoring `circleci/test-batch"`. Friday afternoon and double negatives, maybe I'm missing something?

Reply:
yeah not ignore all of them and not ignore the one specific... idk, eslint is weird

Reply:
We need to switch to biome anyway

Reply:
haha fair enough. I would've thought files were included by default.. But I'm sure there's reasons... Just wanted to confim it wasn't a mistake